# nav_sim.yaml (custom config read by yaml-cpp)
frames:
  map: "map"
  odom: "odom"
  base: "base_link"

topics:
  odom: "/odom"
  cmd_vel: "/cmd_vel"
  goal: "/goal_pose"
  path: "/global_path"
  trajectory: "/trajectory"

smoothing:
  max_skip: 1

trajectory:
  speed: 1.0
  lookahead_distance: 0.7

pure_pursuit:
  max_v: 1.0
  max_w:  1.7
  slowdown_dist: 3.0
  pos_tol: 0.45
  advance_tol: 0.45
  preserve_curvature_on_w_saturation: true
  min_v_near_goal: 0.15